integration_tests/androidx_test/src/test/java/org/robolectric/integrationtests/axt/MissingClassInterceptorTests.java

```diff
@@ -0,0 +1,65 @@
+package org.robolectric.integrationtests.axt;
+
+import static org.junit.Assert.fail;
+
+import androidx.annotation.NonNull;
+import androidx.biometric.BiometricPrompt;
+import androidx.biometric.BiometricPrompt.PromptInfo;
+import androidx.fragment.app.FragmentActivity;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import java.security.NoSuchAlgorithmException;
+import java.util.concurrent.Executor;
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.android.controller.ActivityController;
+
+/** Test intercepting classes not present in OpenJDK. */
+@RunWith(AndroidJUnit4.class)
+public class MissingClassInterceptorTests {
```

Comment:
What about renaming to `BiometricPromptTest` or `CryptoObjectTest`?

Reply:
My idea was to add a placeholder for other similar tests. But since we only have one...

Reply:
@utzcoz Do you want a different comment on top of the test class? If so, what?